packages/next/test/fixtures/00-app-dir-actions/app/client/static/[dynamic-static]/page.js

```diff
@@ -0,0 +1,21 @@
+'use client';
+
+import { useState } from 'react';
+import { increment } from '../../../actions';
+
+export const dynamic = 'force-static';
```

Comment:
This has no effect in `use client` is this being statically generated like we're hoping?